### ISSUE TITLE
Can't login with administrator account

### ISSUE THREAD

**Issue by arkabase** [NodeBB/NodeBB#12185]
### NodeBB version

v3.5.1

### NodeBB git hash

4c543488719cbe6d8716a7dff5af9100945deb6c

### NodeJS version

v20.9.0

### Installed NodeBB plugins

    * nodebb-plugin-2factor@7.4.0 (installed, disabled)
    * nodebb-plugin-composer-default@10.2.24 (installed, enabled)
    * nodebb-plugin-dbsearch@6.2.2 (installed, disabled)
    * nodebb-plugin-emoji@5.1.13 (installed, enabled)
    * nodebb-plugin-emoji-android@4.0.0 (installed, enabled)
    * nodebb-plugin-markdown@12.2.5 (installed, enabled)
    * nodebb-plugin-mentions@4.3.7 (installed, enabled)
    * nodebb-plugin-ntfy@1.7.3 (installed, disabled)
    * nodebb-plugin-spam-be-gone@2.2.0 (installed, disabled)
    * nodebb-rewards-essentials@1.0.0 (installed, enabled)
    * nodebb-theme-harmony@1.1.91 (installed, enabled)
    * nodebb-theme-lavender@7.1.5 (installed, disabled)
    * nodebb-theme-peace@2.1.23 (installed, disabled)
    * nodebb-theme-persona@13.2.42 (installed, disabled)
    * nodebb-widget-essentials@7.0.14 (installed, enabled)

### Database type

MongoDB

### Database version

v7.0.3

### Exact steps to cause this issue

1. Install and setup NodeBB as in the documentation
2. create initial administrrator account
3. restart NodeBB
4. login with administrator account

### What you expected

The user is logged in and has access to NodeBB admin dashboard

### What happened instead

Error message saying identification certificate is invalid.

- Tried multiple times, same result
- Changed password using NodeBB CLI, retry, same result
- Created a new user account, made user admin using NodeBB CLI : user can login but is denied access to admin dashboard.

### Anything else?

I can't understand where the user accounts are saved. My Mongo DB has only 2 collections "objects" and "sessions" and none contains user data. Where is NodeBB data stored ???

**Comment by julianlam** [comment]
In objects.

Can you show us the error, either copy pasted or screenshot?

**Comment by arkabase** [comment]
I finally succeeded to login after changing the password using the password reset email link and the same password as before. Seems that the CLI doesn't set the password correctly, maybe using a wrong secret ?

**Comment by barisusakli** [comment]
Seems like trying to reset password via cli crashes, 
```
TypeError: Cannot read properties of undefined (reading 'get')
    at D:\github\NodeBB\src\user\auth.js:66:38
    at node:internal/util:411:7
    at new Promise (<anonymous>)
    at node:internal/util:397:12
    at User.auth.revokeSession (D:\github\NodeBB\src\user\auth.js:131:28)
    at Object.wrapperCallback [as revokeSession] (D:\github\NodeBB\src\promisify.js:46:11)
    at D:\github\NodeBB\src\user\auth.js:148:42
    at Array.map (<anonymous>)
    at D:\github\NodeBB\src\user\auth.js:148:23
```

Will get it fixed for 3.5.2
